.circleci/config.yml

```diff
@@ -125,6 +134,7 @@ workflows:
       - flake8-tests-on-python
       - flake8-tests-on-legacy-python
       - unit-tests
+      - frontend-unit-tests
```

Comment:
I think it would make sense to have a consistent naming scheme for the jobs, e.g.

- py2-flake8-tests
- py3-flake8-tests
- backend-unit-tests
- frontend-unit-tests

WDYT?

Reply:
Good idea. I will update.

Reply:
Updated. I ended up having a bit more verbose names for the flake8 tests.